x-pack/platform/test/api_integration_deployment_agnostic/apis/streams/conflicts.ts

```diff
@@ -16,55 +22,165 @@ import {
+          forkStream(apiClient, 'logs', stream1),
+          forkStream(apiClient, 'logs', stream2),
+        ]);
+        // Assert than one of the requests failed with a conflict error and the other succeeded
+        // It needs to check either way (success or failure) because the order of execution is not guaranteed
+        expect(responses).to.have.length(2);
+        const successResponse = responses.find(
+          (response) => response.status === 'fulfilled' && response.value.acknowledged
+        );
+        const conflictResponse = responses.find(
+          (response) =>
+            response.status === 'rejected' &&
+            String(response.reason).toLowerCase().includes('conflict')
+        );
+        expect(successResponse).to.not.be(undefined);
+        expect(conflictResponse).to.not.be(undefined);
+      });
     });
 
-    it('should not allow multiple requests manipulating streams state at once', async () => {
-      const stream1 = {
-        stream: {
-          name: 'logs.nginx',
-        },
-        if: {
-          field: 'resource.attributes.host.name',
-          operator: 'eq' as const,
-          value: 'routeme',
-        },
-      };
-      const stream2 = {
-        stream: {
-          name: 'logs.apache',
-        },
-        if: {
-          field: 'resource.attributes.host.name',
-          operator: 'eq' as const,
-          value: 'routeme2',
-        },
-      };
-      const responses = await Promise.allSettled([
-        forkStream(apiClient, 'logs', stream1),
-        forkStream(apiClient, 'logs', stream2),
-      ]);
-      // Assert than one of the requests failed with a conflict error and the other succeeded
-      // It needs to check either way (success or failure) because the order of execution is not guaranteed
-      expect(responses).to.have.length(2);
-      const successResponse = responses.find(
-        (response) => response.status === 'fulfilled' && response.value.acknowledged
-      );
-      const conflictResponse = responses.find(
-        (response) =>
-          response.status === 'rejected' &&
-          String(response.reason).toLowerCase().includes('conflict')
-      );
-      expect(successResponse).to.not.be(undefined);
-      expect(conflictResponse).to.not.be(undefined);
+    describe('classic/wired conflicts', function () {
+      before(async () => {
+        apiClient = await createStreamsRepositoryAdminClient(roleScopedSupertest);
+        await enableStreams(apiClient);
+        const doc = {
+          message: '2023-01-01T00:00:10.000Z error test',
+        };
+        const response = await indexDocument(esClient, 'logs.nginx', doc);
+        expect(response.result).to.eql('created');
+      });
+
+      after(async () => {
+        await esClient.indices.delete({ index: 'logs.nginx' });
+        await disableStreams(apiClient);
+      });
+
+      it('should not allow to create a wired stream with the same name as an existing classic stream', async () => {
```

Comment:
This tests the existing index case not the classic stream, the logic branches depending on the GET data_stream response it might be worth testing both cases